org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLFormattingOptions.java

```diff
@@ -200,6 +200,15 @@ public String getQuotations() {
 		}
 	}
 
+	/**
+	 * Returns the actual quotation value as a String.
+	 * 
+	 * Either a {@code '} or {@code "}
+	 */
+	public String getQuotationAsString() {
+		return getQuotations().equals(XMLFormattingOptions.DOUBLE_QUOTES_VALUE) ? "\"" : "\'";
```

Comment:
good practice is to always put the constant on the LHS of the equality, to avoid potential NPEs:
     
       XMLFormattingOptions.DOUBLE_QUOTES_VALUE. equals(getQuotations())